ifrs17/Utils/ActivityLog.ipynb

```diff
@@ -150,7 +150,7 @@
         {
             "cell_type": "code",
             "source": [
-                "public abstract record KeyedImportExport : KeyedRecord",
+                "public record IOContent : KeyedRecord",
                 "\n{",
```

Comment:
Do we really want to see the table of IO content? We have separate tables of imports and exports that we do not remove. Hence I believe this record must be abstract.

Reply:
@andrey-katz-systemorph I was under the impression that the a new collective table (IOContent) should replace the following:

ExportFile, ImportFile, ImportDataSet, ImportString, ImportStream

and contain all their columns. 

Making it non-abstract actually drops these 5 tables from the DB and creates a new one with the combined information.

Reply:
Then I do not understand this PR at all. You are still keeping all these record, so these tables will be formed, like or dislike. Moreover, from reading the code you populate *these tables* rather than your IOContent. Also you have abstract KeyedImport that inherits from non-abstract IOContent, which looks unusual (not sure whether it is technically allowed, but it breaks the logical hierarchy of the records.)

Reply:
To test if these changes have the expected behavior I used the `Scaffold` notebook and replaced the contents of `Initial` to have the new tables in the DB (+IOContent +IOActivity -ImportExportActivity -ImportFiles -ImportString -ImportStream -ImportDataSet). 

Then I Initialized the DB and IOContent table was populated:
![image](https://user-images.githubusercontent.com/114566025/233590781-79352641-c9c3-4553-9912-783b77c00d1e.png)

I didn't push yet here the updated migration notebooks.

Nevertheless I will try to implement your requested changes because I agree that the logical hierarchy currently is not ideal @andrey-katz-systemorph.

Reply:
@andrey-katz-systemorph No, the tables will not be created, rather there will be 1 table only in the DB, as this is the concrete record. I think this is what we want.

Reply:
abstract inheritance in the middle should also be ok, as abstract is only thereto avoid code repetition.